src/components/skeletonView/index.tsx

```diff
@@ -203,9 +203,16 @@ class SkeletonView extends Component<InternalSkeletonViewProps, SkeletonState> {
       console.error(`RNUILib SkeletonView's requires installing "react-native-linear-gradient" dependency`);
     } else if (_.isUndefined(createShimmerPlaceholder)) {
       console.error(`RNUILib SkeletonView's requires installing "react-native-shimmer-placeholder" dependency`);
-    } else {
+    } else if (ShimmerPlaceholder === undefined) {
       ShimmerPlaceholder = createShimmerPlaceholder(LinearGradient);
     }
+
+    const accessibilityProps = extractAccessibilityProps(this.props);
+    if (props.template === Template.LIST_ITEM) {
+      this.listItemAccessibilityProps = this.getAccessibilityProps('Loading list item', accessibilityProps);
+    } else {
+      this.contentAccessibilityProps = this.getAccessibilityProps('Loading content', accessibilityProps);
+    }
```

Comment:
Why not have it so that `getAccessibilityProps` method is responsible for all things related to accessibility props and move `extractAccessibilityProps` to there?